k2/csrc/properties_test.cc

```diff
@@ -18,6 +18,131 @@ namespace k2 {
 // TODO(haowen): create Fsa examples in a more elegant way (add methods
 // addState, addArc, etc.) and use Test Fixtures by constructing
 // reusable FSA examples.
+TEST(Properties, IsNotValid) {
+  // fsa should contains at least two states.
```

Comment:
should **contain**,  no `-s`

Reply:
updated it https://github.com/danpovey/k2/pull/17